### DISCUSSION TITLE
Help with codemagic.yaml

### DISCUSSION THREAD

**Discussion by ObjetivaSoftware** [codemagic-ci-cd/codemagic-docs#1276] [Q&A] [answered]
Hi Devs,
I'm using the codemagic.yaml file to publish my app in the apple store, but in the end i got the following error:

```
Associated error: The provided entity is missing a required attribute - You must provide a value for the attribute 'whatsNew' with this request
```

So, with this error, my bundle appears in the apple store but is not added to review.
 
I'm kinda sure that this attribute is the "What's New in This Version" field that is required in the apple store.

Can i configure this field in the codemagic.yaml file? Didn't find anything about this anywhere.

**Comment by K-Codemagic** [comment, marked as answer]
Hi,
Please pass the "whatsNew" parameter in your JSON payload.

e.g
`[ { "language": "en-US", "text": "Added new menus", "whatsNew": "Added new menus", } ]`

Feel free to let us know if you have any questions.

**Reply by ObjetivaSoftware** [reply]
Sorry, but which JSON payload?

The only JSON payload i used in this process was the DATA sended to the codemagic API (POST : https://api.codemagic.io/builds) to start my build, where i tryied to add the "whatsNew" but did not worked.

Inside the codemagic's workflow theres no field to input this kind of thing neither.

**Reply by K-Codemagic** [reply]
I meant release_notes.json. Check [this](https://docs.codemagic.io/yaml-notification/publish-release-notes/) out.
